### ISSUE TITLE
Question about the hyperparameters setting

### ISSUE THREAD

**Issue by KcAcoZhang** [xyjigsaw/CENET#2]
Hello, a wonderful job. I want to ask that the hyperparameters settings are the same for all datasets or not.

**Comment by xyjigsaw** [comment]
In our original experiments, $\lambda$ and $\alpha$ are exactly the same. We use hard mode for YAGO and WIKI, soft mode for event-based TKGs.

For data sets with less training data, the dropout value in Oracle can be slightly increased for better performance.

If there are still other problems, please inform us as soon as possible. Your suggestions can be of help to improve our last version of CENET.
